lib/schema/deviceMeta.js

```diff
@@ -30,17 +30,28 @@ var UNKNOWN_PREV = 'status/unknown-previous';
 var idFields = ['type', 'subType', 'time', 'deviceId'];
 schema.registerIdFields('deviceMeta', idFields);
 
-var suspendedReasons = {
-  'manual': true,
-  'low_glucose': true,
-  'alarm': true
-};
-
-var resumedReasons = {
+var statusReasons = {
```

Comment:
The reason codes *weren't* the same before - we (I) simplified them recently because one of them had a tri-partite distinction that wasn't being used. And sure, can make the changes as suggested.